Sources/Charcoal/CharcoalViewController.swift

```diff
@@ -57,6 +57,11 @@ public class CharcoalViewController: UINavigationController {
 
     // MARK: - Init
 
+    public override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+        selectionStore.delegate = self
```

Comment:
I think we can make `selectionStore` into lazy var and set delegate in closure. Then we don't have to override this init and will not forget to set delegate.